### ISSUE TITLE
Error on 0.13.0-beta2

### ISSUE THREAD

**Issue by nate-kennedy** [tfutils/tfenv#192]
Terraform may have depricated the `-version` flag in this version. Attempting to use 0.13.0-beta2 results in the following message:

```
$ tfenv use 0.13.0-beta2
Switching default version to v0.13.0-beta2
Error parsing command-line flags: flag provided but not defined: -version

'terraform --version' failed. Something is seriously wrong
```

**Comment by Zordrak** [comment]
https://github.com/hashicorp/terraform/issues/25276

It wasn't intentional. However leaving this issue open as a change to tfenv from using the -version to the version command will allow tfenv to continue to support 0.12.0-beta2

**Comment by Zordrak** [comment]
Resolved by #191
